_inc/client/state/jetpack-notices/reducer.js

```diff
@@ -14,15 +14,23 @@ import {
 	JETPACK_NOTICES_DISMISS_FAIL,
 	JETPACK_NOTICES_DISMISS_SUCCESS,
 	DISCONNECT_SITE_SUCCESS,
-	RESET_OPTIONS_SUCCESS
+	RESET_OPTIONS_SUCCESS,
+	JUMPSTART_ACTIVATE_SUCCESS,
+	JUMPSTART_SKIP
 } from 'state/action-types';
 import restApi from 'rest-api';
 
 const status = ( state = false , action ) => {
 	switch ( action.type ) {
-		case DISCONNECT_SITE_SUCCESS:
+		case DISCONNECT_SITE_SUCCESS :
```

Comment:
We don't have spaces in [other](https://github.com/Automattic/jetpack/blob/master/_inc/client/state/jumpstart/reducer.js#L27) [similar](https://github.com/Automattic/jetpack/blob/master/_inc/client/state/connection/reducer.js#L28) [reducers](https://github.com/Automattic/jetpack/blob/master/_inc/client/state/modules/reducer.js#L34).
Lines 28 and 31 below should also be updated.